tictactoe/tictactoe.py

```diff
@@ -0,0 +1,116 @@
+
+ls = ['1','2','3','4','5','6','7','8','9']
```

Comment:
Instead of using 1 dimensional list can use two dimensional list using list of list
[[1,2,3][4,5,6],[7,8,9]]